packages/common/src/webrtc/Dialog.ts

```diff
@@ -160,7 +160,7 @@ export default class Dialog {
   }
 
   handleMessage(msg: any) {
-    const { method, params } = msg
+    const { method, params, node_id} = msg
```

Comment:
If you use the `setter` method on Dialog (see comment below) you don't need `node_id` here.